src/executor/evaluate/mod.rs

```diff
@@ -260,6 +261,44 @@ async fn evaluate_function<'a, T: 'static + Debug>(
             Nullable::Null => Ok(Value::Null),
         }
         .map(Evaluated::from),
+
+        Function::Sqrt(expr) => match eval_to_float(expr).await? {
+            Nullable::Value(v) => Ok(Value::F64(v.sqrt())),
+            Nullable::Null => Ok(Value::Null),
+        }
+        .map(Evaluated::from),
+
+        Function::Power { expr, power } => {
+            let number = match eval(expr).await?.try_into()? {
+                Value::F64(number) => number,
+                Value::I64(number) => number as f64,
+                Value::Null => {
+                    return Ok(Evaluated::from(Value::Null));
+                }
+                _ => {
+                    return Err(EvaluateError::FunctionRequiresFloatOrIntegerValue(
+                        "POWER".to_owned(),
+                    )
+                    .into());
+                }
+            };
+            let power = match eval(power).await?.try_into()? {
+                Value::F64(power) => power,
+                Value::I64(power) => power as f64,
+                Value::Null => {
+                    return Ok(Evaluated::from(Value::Null));
+                }
+                _ => {
+                    return Err(EvaluateError::FunctionRequiresFloatOrIntegerValue(
+                        "POWER".to_owned(),
+                    )
+                    .into());
+                }
+            };
```

Comment:
It looks like that you can also use `eval_to_float` in here.

ref. this example uses `eval_to_integer` but it would be good to look around.
https://github.com/gluesql/gluesql/blob/dfc650ccee01e445b9322f4104ad590cb8f1876b/src/executor/evaluate/mod.rs#L459-L474

Reply:
Thanks for advice. I modified it.